app/assets/stylesheets/application.scss

```diff
@@ -8,6 +8,12 @@
 @import 'import-preview';
 @import 'forms';
 
+// FIX: govuk_admin_template seems to be out of sync with the guidance
+// `.error-message` should be removed once govuk_admin_template is updated.
```

Comment:
ok, cool.

Could you please open a PR on `govuk_admin_template` in order to add that? We can then update this once that's merged (in another PR, no need to wait for that to me merged)